vendor.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/coreos/etcd v3.3.25+incompatible // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.1
 	github.com/creack/pty v1.1.11
+	github.com/docker/cli-docs-tool v0.3.0
```

Comment:
I recall we were discussing to make the `docs` bit a separate module, so that this doesn't end up as dependency; did that complicate things?

Reply:
Ah yes indeed I remember now. Sure should be fine with a sub go.mod

Reply:
Ok got something workind without impacting root dependencies: https://github.com/docker/cli/pull/3381/files#diff-0918773ad63e372994befb5ccfe248db86758cd788a248043fb43b9f6abab830R17-R22

That's a bit hacky because we can't use the fully fledged go modules pattern. We can drop that kind of behavior if we switch to go modules in the repo and simply use:

```
module github.com/docker/cli/docs

go 1.16

require (
	github.com/docker/cli v0.0.0+incompatible
	github.com/docker/cli-docs-tool v0.3.0
)

replace github.com/docker/cli v0.0.0+incompatible => ../
```

Reply:
I cna do the same for manpages

Reply:
Yes, that approach works for me (as nobody is expected to use the `docker/cli/docs` package as a dependency

Reply:
Should be ok for manpages